ui/src/translations/fr.json

```diff
@@ -84,5 +84,8 @@
   "no": "Non",
   "volume_creation": "Création du volume",
   "volume_creation_success": "Le volume {name} a été créé avec succèss",
-  "volume_creation_failed": "La création du volume {name} a échoué."
+  "volume_creation_failed": "La création du volume {name} a échoué.",
+  "volume_name_error": "Le nom doit être composé de caractères minuscules, nombres ou de tiret.",
```

Comment:
- `caractères minuscules` => `minuscules` (otherwise we may think you talk about very small characters, it's a bit ambiguous :p)
- `nombres` => `chiffres` (there is a distinction between those two terms in French).